app/helpers/provider/admin/cms/email_templates_helper.rb

```diff
@@ -6,7 +6,7 @@ def disable_sending_button
 
     link_to "Disable Sending Snippet", '#',
                      onclick: js,
-                     class: 'dangerous-button',
+                     class: 'pf-c-button pf-m-secondary pf-m-danger',
```

Comment:
Changed from fully red to this "secondary":

![image](https://github.com/user-attachments/assets/79fe402e-12ff-43c8-b08b-aa56e990f7e6)

Basically because the action is not that discruptive. It just adds a snippet to the template, but unless the updated template is saved, nothing changes.